toonz/sources/toonz/mainwindow.cpp

```diff
@@ -1878,12 +1895,15 @@ void MainWindow::defineActions() {
 
   createMenuWindowsAction(MI_ResetRoomLayout, tr("&Reset to Default Rooms"),
                           "");
-
+  createMenuWindowsAction(MI_MaximizePanel, tr("Toggle Maximize Panel"), "`");
+  createMenuWindowsAction(MI_FullScreenWindow,
+                          tr("Toggle Main Window's Full Screen Mode"),
```

Comment:
What about changing the existing full screen to "Toggle Full Screen Canvas" so this can be "Toggle Full Screen"?